azure/azuredeploy.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "environmentName": {
+      "type": "string"
+    },
+    "keyVaultName": {
+      "type": "string"
+    },
+    "storageAccountName": {
+      "type": "string"
+    },
+    "keyVaultReaderServicePrincipalId": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "location": "[resourceGroup().location]",
+    "tags": {
+      "Environment": "[parameters('environmentName')]",
+      "Parent Business": "Teacher Training and Qualifications",
+      "Portfolio": "Early Years and Schools Group",
+      "Product": "Database of Qualified Teachers",
+      "Service": "Teacher Training and Qualifications",
+      "Service Line": "Teaching Workforce",
+      "Service Offering": "Database of Qualified Teachers"
+    },
+    "tenantId": "9c7d9dd3-840c-4b3f-818e-552865082e16",
+    "tfstateContainerName": "dqtapi-tfstate",
+    "deliveryTeamGroupId": "ce5793a4-b822-4018-a65d-6c04e96918b2"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "2019-06-01",
+      "name": "[parameters('storageAccountName')]",
+      "location": "[variables('location')]",
+      "tags": "[variables('tags')]",
+      "sku": {
+        "name": "Standard_LRS"
+      },
+      "kind": "StorageV2",
+      "properties": {},
+      "resources": [
+        {
+          "type": "blobServices/containers",
+          "apiVersion": "2019-06-01",
+          "name": "[concat('default/', variables('tfstateContainerName'))]",
+          "dependsOn": [
+            "[concat('Microsoft.Storage/storageAccounts/', parameters('storageAccountName'))]"
+          ]
+        }
+      ]
+    },
+    {
+      "type": "Microsoft.KeyVault/vaults",
+      "apiVersion": "2019-09-01",
+      "name": "[parameters('keyVaultName')]",
+      "location": "[variables('location')]",
+      "tags": "[variables('tags')]",
+      "properties": {
+        "accessPolicies": [
+          {
+            "tenantId": "[variables('tenantId')]",
+            "objectId": "[variables('deliveryTeamGroupId')]",
+            "permissions": {
+              "secrets": [
+                "get",
+                "list",
+                "set",
+                "delete",
+                "recover",
+                "backup",
+                "restore"
+              ]
+            }
+          },
+          {
+            "tenantId": "[variables('tenantId')]",
+            "objectId": "[parameters('keyVaultReaderServicePrincipalId')]",
+            "permissions": {
+              "secrets": [
```

Comment:
Shouldn't we leave only readonly policies?

Reply:
Amended to have the `get` and `list` permissions only.